.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,68 @@
+---
+name: '🚀 Release'
+about: For internal use only - issue template for release tasks
+title: ''
```

Comment:
Should this include a placeholder title? e.g. "Release GOV.UK Frontend vX.Y.Z"

Reply:
Ohh, that's a good one, didn't think of that, I'll amend!